src/components/tl/mlx5/tl_mlx5_team.c

```diff
@@ -164,50 +165,46 @@ ucc_status_t ucc_tl_mlx5_team_create_test(ucc_base_team_t *team)
     return status;
 }
 
-ucc_status_t ucc_tl_mlx5_coll_init(ucc_base_coll_args_t *coll_args,
-                                   ucc_base_team_t *     team,
-                                   ucc_coll_task_t **    task_h)
-{
-    ucc_status_t status = UCC_OK;
-
-    switch (coll_args->args.coll_type)
-    {
-    case UCC_COLL_TYPE_BCAST:
-        status = ucc_tl_mlx5_bcast_mcast_init(coll_args, team, task_h);
-        break;
-    default:
-        status = UCC_ERR_NOT_SUPPORTED;
-    }
-
-    return status;
-}
-
 ucc_status_t ucc_tl_mlx5_team_get_scores(ucc_base_team_t *  tl_team,
                                          ucc_coll_score_t **score_p)
 {
-    ucc_tl_mlx5_team_t *team = ucc_derived_of(tl_team, ucc_tl_mlx5_team_t);
-    ucc_base_context_t *ctx  = UCC_TL_TEAM_CTX(team);
-    ucc_memory_type_t   mt   = UCC_MEMORY_TYPE_HOST;
-    ucc_coll_score_t *  score;
-    ucc_status_t        status;
+    ucc_tl_mlx5_team_t *team  = ucc_derived_of(tl_team, ucc_tl_mlx5_team_t);
+    ucc_base_context_t *ctx   = UCC_TL_TEAM_CTX(team);
+    ucc_base_lib_t     *lib   = UCC_TL_TEAM_LIB(team);
+    ucc_memory_type_t   mt[2] = {UCC_MEMORY_TYPE_HOST, UCC_MEMORY_TYPE_CUDA};
```

Comment:
@Sergei-Lebedev should UCC_MEMORY_TYPE_CUDA_MANAGED be added here as well once PR #801 is merged?